MakeHist/mumu.C

```diff
@@ -0,0 +1,337 @@
+#define mumu_cxx
+#include "mumu.h"
+#include <TH2.h>
+#include <TStyle.h>
+#include <TCanvas.h>
+#include <TVector2.h>
+#include <vector>
+#include <TH1.h>
+
+struct Lepton{
+    TLorentzVector LV;
+    float Iso;
+};
+
+
+void mumu::Loop(TString sample_name)
+{
+//   In a ROOT session, you can do:
+//      root> .L mumu.C
+//      root> mumu t
+//      root> t.GetEntry(12); // Fill t data members with entry number 12
+//      root> t.Show();       // Show values of entry 12
+//      root> t.Show(16);     // Read and show values of entry 16
+//      root> t.Loop();       // Loop on all entries
+//
+
+//     This is the loop skeleton where:
+//    jentry is the global entry number in the chain
+//    ientry is the entry number in the current Tree
+//  Note that the argument to GetEntry must be:
+//    jentry for TChain::GetEntry
+//    ientry for TTree::GetEntry and TBranch::GetEntry
+//
+//       To read only selected branches, Insert statements like:
+// METHOD1:
+//    fChain->SetBranchStatus("*",0);  // disable all branches
+//    fChain->SetBranchStatus("branchname",1);  // activate branchname
+// METHOD2: replace line
+//    fChain->GetEntry(jentry);       //read all branches
+//by  b_branchname->GetEntry(ientry); //read only this branch
+   if (fChain == 0) return;
+
+   //Create root file with histograms
+   TFile *output = new TFile(sample_name+"_DiMu_Nobjet_Histograms.root","recreate");
+
+   //histograms
+   TH1F *run_hist = new TH1F("run_hist","run; Total number of events passed through the pre-selection cut",3,0,3); run_hist->Sumw2();
+   TH1F *leppT_hist = new TH1F("leppT_hist","lepton pt",100,0,500); leppT_hist->Sumw2();
```

Comment:
I suggest adding axis descriptions to all histograms. TH1F("name","title;x-axis titles;y-axis title")